assets/docs/bohemian-visual-recognition-alliance/models/mushroom-identification_v1/1.md

```diff
@@ -28,10 +28,15 @@ This model takes input of images.
 The model can be loaded in a Python script as follows:
 
 ```python
-module = hub.Module("https://tfhub.dev/bohemian-visual-recognition-alliance/models/mushroom-identification_v1/1")
-height, width = hub.get_expected_image_size(module)
-images = ...  # A batch of images with shape [batch_size, height, width, 3].
-output_dict = module(images)  # Output dictionary.
+model = hub.load("https://tfhub.dev/bohemian-visual-recognition-alliance/models/mushroom-identification_v1/1")
+
+@tf.function(input_signature=[tf.TensorSpec(shape=[None, None, None, 3], dtype=tf.float32)])
+def call_model(image_tensor):
+  output_dict = module.signatures["serving_default"](image_tensor)
+  return output_dict['out']
+  
+images = ...  # A batch of images with shape [batch_size, 360, 360, 3].
```

Comment:
I think 360x360 is the image size for the v2 model, this one specifies 500x500 in the documentation but the input shape of its serving_default signature is 299x299.

Reply:
I put it back to what it was before height, width. Actually the current code "works" with other sizes as well, I assume by truncation or reshaping.